src/api/spec/factories/packages.rb

```diff
@@ -89,16 +89,17 @@
 
     factory :package_with_changes_file do
       transient do
-        changes_file_content '
--------------------------------------------------------------------
-Fri Aug 11 16:58:15 UTC 2017 - tom@opensuse.org
+        changes_file_content do
+          '
+        -------------------------------------------------------------------
+        Fri Aug 11 16:58:15 UTC 2017 - tom@opensuse.org
 
-- Testing the submit diff
+        - Testing the submit diff
 
--------------------------------------------------------------------
-Wed Aug  2 14:59:15 UTC 2017 - iggy@opensuse.org
-
-- Temporary hack'
+        -------------------------------------------------------------------
+        Wed Aug  2 14:59:15 UTC 2017 - iggy@opensuse.org
+        - Temporary hack'
+        end
```

Comment:
Nitpick: Changes of indentation not needed and not related to the pull request. But I don't care if the don't change the result of the tests.

Reply:
This was autocorrected and it doesn't affect the result of the tests.